example.env

```diff
@@ -1,4 +0,0 @@
-BASE_URL=http://localhost:3004
```

Comment:
hi Kevin, you may want to keep the `example.env` file for people who may want to clone the project later